examples/notifications.py

```diff
@@ -0,0 +1,21 @@
+from sseclient import SSEClient
+from httplib import IncompleteRead
```

Comment:
Please include a header and example usage like the other examples in this directory.